.gitignore

```diff
@@ -1,4 +1,7 @@
 **/*
 !**/*.go
 !**/
-!.gitignore
\ No newline at end of file
+!.gitignore
+!Gopkg.lock
+!Gopkg.toml
+/vendor
```

Comment:
Thank you for keeping web clean. 
But I'm afraid, it was a part of a task to add vendor folder to repo.

Reply:
i thought that `Gopkg.lock` and `Gopkg.toml` is enough to make sure that i actually run scripts...